CMakeLists.txt

```diff
@@ -44,6 +44,7 @@ endif()
 
 if(WIN32)
   option(XSENS_MVN_USE_SDK "Build the driver and the wrapper for the real MVN system using the MVN SDK" ON)
+  add_compile_definitions(_SILENCE_EXPERIMENTAL_FILESYSTEM_DEPRECATION_WARNING)
```

Comment:
I would add a comment here to specify the reason for this definition

Reply:
this is line is added to suppress the following error while compiling that is caused by https://github.com/robotology/wearables/blob/master/XSensMVN/src/XSensMVNDriverImpl.cpp#L12

```
The <experimental/filesystem> header providing std::experimental::filesystem is dep
recated by Microsoft and will be REMOVED. It is superseded by the C++17 <filesystem> header providing std::filesystem.
You can define _SILENCE_EXPERIMENTAL_FILESYSTEM_DEPRECATION_WARNING to acknowledge that you have received this warning.

``` 

@lrapetti Updated in https://github.com/robotology/wearables/commit/9cffe091cdcc2980deb91c85b3c09434309ea8fb

Reply:
This may break in any future version of MSVC. Probably it could make sense to instead detect if msvc is used, and in that case include directly `<filesystem>` .

Reply:
@GiulioRomualdi I guess this discussion is relative to the compilation problem you encounter.